gui/src/app/view/settings.rs

```diff
@@ -533,8 +534,13 @@ pub fn rescan<'a>(
                                 .spacing(10),
                         )
                         .push_maybe(if invalid_date {
+                            Some(p1_regular("Provided date is invalid").style(color::RED))
+                        } else {
+                            None
+                        })
+                        .push_maybe(if past_possible_height {
                             Some(
-                                p1_regular("Provided date is either invalid or anterior to the genesis block's timestamp.")
+                                p1_regular("Provided date earlier than the node prune height")
```

Comment:
I also get this error when passing a future date. Perhaps the message could include both possibilities.